catalog/ui/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -78,6 +78,7 @@ define([
                 }).map(function(metacardType){
                     return {
                         label: metacardType.alias || metacardType.id,
+                        description: (properties.attributeDescriptions || {})[metacardType.id],
```

Comment:
So the description tooltip ends up as either the user entered description or the `metacardType.id`?

Reply:
`metacardType.id` is the lookup key. If there are no attribute descriptions, or there isn't one for a particular metacard id, this will return `undefined`.

Reply:
Ah, makes sense.